app/src/main/java/io/github/droidkaigi/confsched2018/di/AppModule.kt

```diff
@@ -24,7 +24,7 @@ import io.github.droidkaigi.confsched2018.util.rx.AppSchedulerProvider
 import io.github.droidkaigi.confsched2018.util.rx.SchedulerProvider
 import javax.inject.Singleton
 
-@Module(includes = [(ViewModelModule::class)])
+@Module
```

Comment:
I want to keep this rule. 👀 
Even now, with some codes we are not following that rule. 🙇  

> When only a single annotation without arguments is present it may be placed on the same line as the declaration.

https://android.github.io/kotlin-guides/style.html#annotations